### ISSUE TITLE
Monetization idea

### ISSUE THREAD

**Issue by infinitytec** [chexbox/AlgCalc#71]
The (old and depreciated) license for this said we might use ads at some point.
I, however, being a person who (occasionally) reads EULAS and privacy policies, am not a big fan of the online advertising model, and do not think this project would be-as intended-truly free with it. So my suggestion: Patreon, or a similar service. We can add links to our personal pages for this. 

Thoughts?

**Comment by chexbox** [comment]
I added that to the license for potential in the future. I don't see us needing money for it in the near future, however we could link to personal sites.

**Comment by infinitytec** [comment]
Yep. I figured that. I just thought of this and wanted to throw it out.
